mgmtsystem_hrn/models/mgmtsystem_hazard_severity.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 - TODAY, Kaynnan Lemes <kaynnan.lemes@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MgmtsystemHazardSeverity(models.Model):
+
+    _inherit = "mgmtsystem.hazard.severity"
+
+    value = fields.Float(required=True)
```

Comment:
nao está muito generico o nome deste campo ?

Reply:
Esses campo value são uma refatoração, eles já existem no módulo inherit e estou transformando eles para um valor float

Reply:
mas qual o impacto se por exemplo alguem não for usar a opcao D para calculo ?

Reply:
Nenhum impacto, mas a ideia que o Douglas passou seria trabalhar com valores float ao invés de inteiro para os calculos de riscos

Reply:
O pessoal pode usar tanto com a formula somente A * B, ou incluir a D

Reply:
acho que rola adicionar comentario em cada campo deste dizendo que está sobreescrevendo e alterando para float.. pra gente nao esquecer disso. oq acha ?